libs/ledger-live-common/src/onboarding/hooks/useOnboardingStatePolling.ts

```diff
@@ -80,7 +120,26 @@ export const useOnboardingStatePolling = ({
     setAllowedError,
     setFatalError,
     stopPolling,
+    getOnboardingStatePolling,
   ]);
 
   return { onboardingState, allowedError, fatalError };
 };
+
+const isDifferentOnboardingState = (
+  prevOnboardingState: OnboardingState | null,
+  newOnboardingState: OnboardingState | null
+): boolean => {
+  // OnboardingState are simple JSON-style objects, without methods
+  return (
+    JSON.stringify(prevOnboardingState) !== JSON.stringify(newOnboardingState)
+  );
+};
+
+const isDifferentError = (
+  prevError: Error | null,
+  newError: Error | null
+): boolean => {
+  // Only interested if the errors are instances of the same Error class
+  return prevError?.constructor !== newError?.constructor;
```

Comment:
Can we try 
```js 
prevError instanceof newError
```

Reply:
`instanceof` needs a "Constructor" on the right side ([doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/instanceof#syntax)). So `prevError instanceof newError` would not work. 

The easiest in my case is just to check the constructor